projects/arlas-toolkit/src/lib/services/collection/arlas-collection.service.ts

```diff
@@ -82,4 +82,13 @@ export class ArlasCollectionService extends BaseCollectionService {
     }
     return false;
   }
+  // Retrieve all collections used in a dashboards
+  public getCollectionFromDashboard(config: any): Set<string> {
+    const flattenedConfig = flattenData(config);
+    const collections = new Set<string>();
+    Object.keys(flattenedConfig)
```

Comment:
There is the exceptional case of MetricsTablesContributor, that has an attribute `configuration`.
The attribute configuration is an array of:
```
export interface MetricsVectorConfig {
    termfield: string;
    collection: string;
    metrics: MetricConfig[];
}
```
and it includes the collection